pallets/staking/src/lib.rs

```diff
@@ -2748,7 +2752,11 @@ impl<T: Trait> Module<T> {
                 nominator_exposure_part * validator_leftover_payout;
             // We can now make nominator payout:
             if let Some(imbalance) = Self::make_payout(&nominator.who, nominator_reward) {
-                Self::deposit_event(RawEvent::Reward(nominator.who.clone(), imbalance.peek()));
+                Self::deposit_event(RawEvent::Reward(
+                    <Identity<T>>::get_identity(&nominator.who).unwrap_or_default(),
+                    nominator.who.clone(),
+                    imbalance.peek(),
+                ));
             }
```

Comment:
Logic is repeated twice. Could we extract this including the `make_payout` bit to a function?

Reply:
I know this is duplicated code but I would rather not do any refactoring here since this is borrowed code. Any refactoring we do now will only cost us time in future when we upgrade to newer versions of the staking pallet from substrate.